s26856-DawidCybuch/zadanie_2.txt

```diff
@@ -68,10 +68,15 @@ Umieścić w nim plik o nazwie strona.html o następującej zawartości:
 Utowrzyłem folder public_html za pomocą komendy mkdir public_html.
 Następnie utworzyłem plik komendą nano strona.html i wpisałem treść podaną w zadaniu.
 Sprawdzam uprawnienia za pomocą komendy ls -l.
-Wykazuje mi że że każdy user może to odczytać r-- to oznacza że każdy może zobaczyć moją stronę, ale nikt inny nie może jej edytować.
+Po wykonaniu komendy ls -l dostaje informacje jakie są przydzielone uprawnienia
+Owner -rw- - czyli jestem w stanie "widziec" oraz edytować ten plik
+Grupa -r-- - Grupa może odczytywać ale nie mogą edytować
+Wszyscy -r-- - Wszyscy mogą odczytać ale nie mogą edytować
 http://szuflandia.pjwstk.edu.pl/~s26856/strona.html
 
-
-
-
-
+ODP.
+Po wykonaniu notacji copy%paste zostalo otrzymalem taki wynik
+Z tego co widze na wszystkich plaszczyznach sa przydzielone takie same uprawnienia
```

Comment:
Nie o to chodziło.
Zastanów się. Jak wchodzisz do budynku uczelni to ile drzwi pokonujesz po drodze?
A w przypadku komputera co jest drzwiami?

Reply:
Tutaj mam problem, poprzez wylistowanie uprawnien ls -l właśnie pojawiają mi sie te rzeczy a stona.html właśnie posiada takie uprawnienia jak podałem w pierwszej odpowiedzi tj. -rw-r--r--

Reply:
/home/PJWSTK/$USER/ - musi mieć minimum 001
/home/PJWSTK/$USER/public_html - minimum 001
/home/PJWSTK/$USER/public_html/strona.html - minimum 004

Tylko nie ja miałem to zadanie wykonać.
Sami mieliście zbadać jakie te minimalne uprawnienia są poprzez sukcesywne odbieranie kolejnych bitów.